src/views/Details/NFTAssetDetails.vue

```diff
@@ -0,0 +1,394 @@
+                        <td class="text-break" v-if="nftAsset.asset_contract">
+                          <span class="text-primary d-flex align-items-center">
+                            {{ shortenAddress(nftAsset.asset_contract.address) }}
+                            <CopyIcon
+                              @click="copy(nftAsset.asset_contract.address)"
+                              class="copy-icon"
+                          /></span>
+                        </td>
+                      </tr>
+                      <tr>
+                        <td class="text-muted text-left small-12" id="your_to_address">Token ID</td>
+                        <td class="text-break" v-if="nftAsset.token_id">
+                          <span class="text-primary">
+                            {{ nftAsset.token_id }}
+                            <CopyIcon @click="copy(nftAsset.token_id)" class="copy-icon"
+                          /></span>
+                        </td>
+                      </tr>
+                      <tr>
+                        <td class="text-muted text-left small-12">Token Standard</td>
+                        <td class="text-break" v-if="nftAsset.asset_contract">
+                          {{ nftAsset.asset_contract.schema_name }}
+                        </td>
+                      </tr>
+                      <tr>
+                        <td class="text-muted text-left small-12">Blockchain</td>
+                        <td class="text-break">
+                          <img :src="getAssetIcon(asset)" class="asset-icon" />
+                          Ethereum
+                        </td>
+                      </tr>
+                    </tbody>
+                  </table>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </template>
+    </div>
+  </div>
+</template>
+
+<script>
+import { mapGetters, mapMutations, mapState } from 'vuex'
+import { shortenAddress } from '@liquality/wallet-core/dist/utils/address'
+import ChevronDownIcon from '@/assets/icons/chevron_down.svg'
+import CopyIcon from '@/assets/icons/copy.svg'
+import SendIcon from '@/assets/icons/send_nft.svg'
+import ShareIcon from '@/assets/icons/share_nft.svg'
+import NavBar from '../../components/NavBar.vue'
+import { chains } from '@liquality/cryptoassets'
+import { getAssetIcon } from '@/utils/asset'
+import { getAccountIcon } from '@/utils/accounts'
+
+export default {
+  data() {
+    return {
+      showFullscreen: false,
+      activeTab: 'overview',
+      asset: 'ETH',
+      nftAsset: null
+    }
+  },
+  components: {
+    ChevronDownIcon,
+    CopyIcon,
+    SendIcon,
+    ShareIcon,
+    NavBar
+  },
+  computed: {
+    ...mapGetters(['accountsData']),
+    ...mapState(['activeNetwork']),
+    source() {
+      return this.$route.fullPath
+    },
+    account() {
+      return this.accountsData.filter((account) => account.chain === 'ethereum')[0]
```

Comment:
Import `ChainId` enum from cryptoassets and use it instead of 'ethereum' string.